workspaces/tech-radar/plugins/tech-radar/src/api.ts

```diff
@@ -44,185 +45,3 @@ export interface TechRadarApi {
    */
   load: (id: string | undefined) => Promise<TechRadarLoaderResponse>;
 }
-
```

Comment:
This mass deletion / movement of these types from tech-radar to tech-radar-common makes this PR a fairly decent break / major release. Any users who imported TechRadarLoaderResponse from tech-radar in their client implementations will need to install tech-radar-common and update that import statement upon upgrading. This is costly, but facilitates zod type checking which is nice. Open to alternatives if folks strongly disagree with that set up - could make it smoother defining the types in two places and doing this delete in a later release.